app/src/main/java/com/einmalfel/podlisten/DownloadReceiver.java

```diff
@@ -297,6 +297,9 @@ public void onReceive(Context context, Intent intent) {
                                 intent.getLongExtra(DownloadManager.EXTRA_DOWNLOAD_ID, 0L));
           updateDownloadQueue(context, false);
           break;
+        default:
+          Log.e(TAG, "switch operator default clause reached");
```

Comment:
Ideally, error messages should give a cue about what went wrong. Like this:
Log.e(TAG, "Unexpected intent received: " + intent);